cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/GetContainerIdOfSegmentCommand.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.segmentstore;
+
+import io.pravega.cli.admin.CommandArgs;
+import io.pravega.client.connection.impl.ConnectionPool;
+import io.pravega.controller.server.SegmentHelper;
+import io.pravega.shared.NameUtils;
+import io.pravega.shared.protocol.netty.WireCommands;
+import io.pravega.shared.segment.SegmentToContainerMapper;
+import com.google.common.base.Preconditions;
+import lombok.Cleanup;
+import org.apache.curator.framework.CuratorFramework;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Gets the container id of the given segment.
+ * The arguments for the command that is the fully qualified segment name.
+ */
+public class GetContainerIdOfSegmentCommand extends SegmentStoreCommand {
+
+    private final SegmentToContainerMapper segToConMapper;
```

Comment:
Nit: Let's use a more understandable naming for this field, such as `segmentToContainerMapper`.

Reply:
I have made the change.